core/builtins/native/kotlin/Any.kt

```diff
@@ -16,10 +16,25 @@
 
 package kotlin
 
+/**
+ * The root of the Kotlin class hierarchy. Every Kotlin class has [Any] as a superclass.
+ */
 public open class Any {
+    /**
+     * Indicates whether some other object is "equal to" this one. Implementations must follow
+     * the same contract as the [Java equals() method](http://docs.oracle.com/javase/7/docs/api/java/lang/Object.html#equals%28java.lang.Object%29).
+     * Note that the `==` operator in Kotlin code is translated into a null-safe call to [equals].
```

Comment:
Expand "null-safe call". I would read it as "a?.equals(b)" which is wrong, because expression type for `==` is `Boolean`, not `Boolean?`